src/Modules/Finance/Header/Header.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { TargetSum } from './TargetSum/TargetSum';
+import { Total } from './Total/Total';
+import { Transaction } from './Transaction/Transaction';
+
+import styles from './Header.module.css';
+
+type IHeaderProps = {
+
+}
+
+type IHeaderState = {
+	targetSumValue: string,
+	targetSumConfirmed: boolean,
+	totalSum: number,
+	currentTransactionValue: string,
+	inputChanged: boolean,
+	transactionConfirmed: boolean,
+	transactor: string
+}
+
+export class Header extends React.Component<IHeaderProps, IHeaderState> {

```

Comment:
Would be great to re-write it as a functional component.

Reply:
You did it, didn't you?

Reply:
Yes, I did it. Check this commit 
b6fb92c